scripts/abi_go_bindings.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -e
+
+source ./scripts/utils.sh
+
+setARCH
+
+TELEPORTER_PATH=$(
+  cd "$(dirname "${BASH_SOURCE[0]}")"
+  cd .. && pwd
+)
+
+DEFAULT_CONTRACT_LIST="TeleporterMessenger ERC20Bridge ExampleCrossChainMessenger BlockHashPublisher BlockHashReceiver"
```

Comment:
For completeness, let's also include `BridgeToken`

Reply:
added `BridgeToken`